site-manage/pageorder/tool/src/webapp/content/css/PageList.css

```diff
@@ -69,6 +69,21 @@ li.sortable_hover
 	padding: 4px;
 	background-color: var(--sakai-background-color-3);
 }
+
+.grab_cursor
+{
+	cursor: grab;
+	cursor:-moz-grab;
+	cursor: -webkit-grab;
+}
+
+.grabbing_cursor 
+{
+    cursor: grabbing;
+    cursor: -webkit-grabbing;
+    cursor: -moz-grabbing;
+} 
```

Comment:
This is the same as above, so just add the two classes to the same selector:

`.grab_cursor, .grabbing_cursor`